rust/main/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -648,10 +648,15 @@ pub struct SealevelMailboxIndexer {
     program_id: Pubkey,
     dispatch_message_log_meta_composer: LogMetaComposer,
     delivery_message_log_meta_composer: LogMetaComposer,
+    advance_log_index: bool,
 }
 
 impl SealevelMailboxIndexer {
-    pub fn new(conf: &ConnectionConf, locator: ContractLocator) -> ChainResult<Self> {
+    pub fn new(
+        conf: &ConnectionConf,
+        locator: ContractLocator,
+        advance_log_index: bool,
```

Comment:
I see some are `advance_log_meta` and some are `advance_log_index`, should we consolidate around one?
also wondering if it should be `advanced_..`

Reply:
Yes, just a misprint. 

Regarding `advance...`. The idea is that relayer relies on a `basic` log meta while scraper relies on `advanced` one.